docs/filesystem_layer/stratum1.md

```diff
@@ -115,25 +115,28 @@ The second time you run it, you should get a cache hit:
 
 ```bash
 X-Cache: HIT from <url-or-ip-to-your-stratum1>
-
 ```
 
-Example with the Norwegian Stratum 1:
+Example with the EESSI Stratum 1 running in AWS:
 
 ```bash
-curl --head http://bgo-no.stratum1.cvmfs.eessi-infra.org/cvmfs/software.eessi.io/.cvmfspublished
+curl --head http://aws-eu-central-s1.eessi.science/cvmfs/software.eessi.io/.cvmfspublished
 ```
 
```

Comment:
And here
```
Verification of the Stratum 1 using a client
```

That way, it is more clear that the following documentation is about _configuring that client_ (and not about configuring the stratum 1 anymore)